src/core/annotation.js

```diff
@@ -603,6 +606,23 @@ class Annotation {
     this.color = getRgbColor(color);
   }
 
+  /**
+   * Set the color for background and border if any.
+   * The default values are transparent.
+   *
+   * @public
+   * @memberof Annotation
+   * @param {Dict}  - The MK dictionary
+   */
+  setBorderAndBackgroundColors(mk) {
+    if (mk instanceof Dict) {
+      this.borderColor = getRgbColor(mk.getArray("BC"), null);
```

Comment:
The borders seem thicker now in the reference tests; it looks like the borders are applied twice now, once here and once in existing CSS perhaps?

Moreover, when opening `annotation-text-widget.pdf` from the test suite and clicking the combs field, the border is now extended behind the field (due to the way padding works for the combs fields). Would it maybe be a solution to make the border, just like the background, transparent on focus?

Reply:
The container and the element had a border and so I just removed the one from the element.
About the focus issue on the comb field, I just removed https://github.com/mozilla/pdf.js/blob/93ed4bfa119fe82c1fe1fe9cd7d08b1ae719b49e/web/annotation_layer_builder.css#L144-L152 which seems to be useless now (something has likely been fixed in Firefox) and without any consequence with the pdf in #12865.